src/components/APIClient.ts

```diff
@@ -0,0 +1,108 @@
+import * as vscode from 'vscode'
+import { InfluxDB, QueryApi } from '@influxdata/influxdb-client'
+import * as InfluxDB1 from 'influx'
+import { BucketsAPI, OrgsAPI, TasksAPI } from '@influxdata/influxdb-client-apis'
+
+import { IInstance, InfluxVersion } from '../types'
+
+// XXX: rockstar (30 Sep 2021) - the following is why we self-medicate.
+// VSCode is an electron app, and a node and electron both ship with their
+// own list of root CAs. Today was the day that a LetsEncrypt root CA expired,
+// and the newer root CA is not bundled in with electron and/or the server side
+// is not fully using the new root CA--I can't be sure of either. For now,
+// including this CA cert in the agent of the request will fix that issue and
+// not block users. It is awful and I will never forgive myself. See also,
+// https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/
+// https://community.letsencrypt.org/t/issues-with-electron-and-expired-root/160991
+// https://github.com/electron/electron/issues/31212
```

Comment:
This is a great comment. When can we remove it? In other words can we add some info about when it will be safe to remove this comment and its associated hack? Are there known conditions we need to watch for? Maybe once VSCode has released a new update?